parser/src/genquery2_sql.cpp

```diff
@@ -141,7 +161,11 @@ namespace
 	// clang-format on
 
 	// clang-format off
+#ifdef IRODS_ENABLE_42X_COMPATIBILITY
+    constexpr auto table_edges = std::array<edge_type, 20>{{
```

Comment:
same.  magic 20?

Reply:
I think this is well-defined. This will go away once 4.2 is dropped.